website_variants_extra/static/src/js/test_comments_qty.js

```diff
@@ -0,0 +1,44 @@
+(function(){
+    'use strict';
+    openerp.Tour.register({
+        id: 'shop_test_comment_qty',
+        name: 'Test Product Name with Variants',
+        path: '/shop/product/ipad-retina-display-5',
+        mode: 'test',
+        steps: [
+            {
+                title:     "Post first comment",
+                onload:    function(Tour){
+                               $('#comment textarea').text('1st Comment');
+                               $('#comment .fa-star-o:last').trigger('click');
+                               $('#comment .btn').trigger('click');
```

Comment:
Only when a form has a `preventDefault` or the action of the button is given by a `.click` event on JS is necessary to trigger the click event, when is a button on a form you must use `element` key to trigger its `post` method, the last trigger here is wrong. please replace with the right `element` key as follows: `element: "form a:contains(Post)",`